tests/integration/transactions/test_account_delete.py

```diff
@@ -25,7 +26,7 @@ def test_all_fields(self):
             destination=DESTINATION.classic_address,
             destination_tag=DESTINATION_TAG,
         )
-        response = submit_transaction(account_delete, WALLET)
+        response = submit_transaction(account_delete, WALLET, JSON_RPC_CLIENT, False)
```

Comment:
The client doesn't need to be passed in here
```suggestion
        response = submit_transaction(account_delete, WALLET, check_fee=False)
```

Reply:
Thanks, I updated that